<launch>
	<arg name="node_name" default="control_fsm_node"/>
	<arg name="param_file" default="control_fsm_default_config.yaml"/>
    	<node name="$(arg node_name)" pkg="control_fsm" type="control_fsm_main" output="screen">
    		<rosparam file="$(find control_fsm)/$(arg param_file)" command="load"/>
    	</node>
</launch>
